flight/uavobjectlib/Makefile

```diff
@@ -66,8 +66,15 @@ LDFLAGS += -Wl,--fatal-warnings
 SRC += $(filter-out $(OPUAVSYNTHDIR)/uavobjectsinit.c,$(wildcard $(OPUAVSYNTHDIR)/*.c))
 NO_AUTO_UAVO = YES
 
+# Backup the environmental variable before entering firmware-common.mk
```

Comment:
This is the one part of this I'm uncomfortable with. It seems very hackish.

Ties in to https://github.com/TauLabs/TauLabs/pull/2209/files#diff-12b5ccf31b114976029c5c17f55b9fc6R9.